connection-service/new-app-gateway/azuredeploy.json

```diff
@@ -238,68 +351,27 @@
         ],
         "urlPathMaps": [
           {
-            "name": "urlPathMap1",
+            "name": "pcoipBrokerProtocolUserInterfacePath",
```

Comment:
broker protocol?

Reply:
The URL path does both PBP if it starts with /pcoip-broker/ and otherwise redirects to the UI

Reply:
Right - so one path map. Got it.